config/interpolate_funcs.go

```diff
@@ -529,6 +530,33 @@ func interpolationFuncSignum() ast.Function {
 	}
 }
 
+// interpolationFuncSort sorts a list of a strings lexographically
+func interpolationFuncSort() ast.Function {
+	return ast.Function{
+		ArgTypes:   []ast.Type{ast.TypeList},
+		ReturnType: ast.TypeList,
+		Variadic:   false,
+		Callback: func(args []interface{}) (interface{}, error) {
+			inputList := args[0].([]ast.Variable)
+
+			// Ensure that all the list members are strings and
+			// create a string slice from them
+			members := make([]string, len(inputList))
+			for i, val := range inputList {
+				if val.Type != ast.TypeString {
+					return nil, fmt.Errorf(
+						"sort() may only be used with lists containg strings.")
```

Comment:
s/containg/containing/

Would be nice if this message printed unexpected type a la `fmt.Errorf("list has non-string element (%T)", value.Type.String())` above.